alacritty/src/config/ui_config.rs

```diff
@@ -85,6 +85,12 @@ pub struct UiConfig {
     /// Bindings for the mouse.
     #[config(deprecated = "use mouse.bindings instead")]
     mouse_bindings: MouseBindings,
+
+    /// Configuration file imports.
+    ///
+    /// This is never read since the field is directly accessed through the config's
+    /// [`toml::Value`], but still present to prevent unused field warnings.
+    xmport: Vec<String>,
```

Comment:
What's the reason to call it `xmport`? Just to clarify that it's dead?

Reply:
Oh that was to test it doesn't work. I forgot to save it when changing it back. :D